eventrouter.h

```diff
@@ -20,6 +20,11 @@
 #include "eventrouter/internal/module.h"
 #include "eventrouter/internal/task_.h"
 
+/// Extensions only available in the baremetal implementation.
+#ifdef ER_BAREMETAL
+#include "eventrouter/internal/eventrouter_baremetal.h"
+#endif
+
```

Comment:
All the original eventrouter functions are supported for the baremetal eventrouter. The baremetal implementation supports a few more functions to replace the "receive from queue" functionality that we got from FreeRTOS.